client/components/download-link/renderers/docx-renderer.js

```diff
@@ -611,17 +632,18 @@ export default (application, sections, values, updateImageDimensions) => {
       case 'permissible-purpose':
         renderPermissiblePurpose(doc, field, value, values);
         break;
-
-      case 'text':
+      case 'text': {
+        renderText(doc, value, noSeparator);
+        if (field.name === 'title') {
+          renderLicenceHolder(doc);
+        }
```

Comment:
This looks like it's going to end up rendering twice.

I'd be tempted to just pass the licenceHolder as a name string from the point where it is inserted into the values, and not need to do any special processing at all in this component.

Reply:
I would give it a custom field type rather than having a special case of `text` though.